examples/manual_enhancement.py

```diff
@@ -0,0 +1,263 @@
+        "enhancement",
+        (-225, 75),
+        207.4e-6,
+        0.959,
+        T1s,
+        1.5154,
+    ),
+]:
+    # {{{ Load and plot raw data
+    outname = "%s_%s" % (date, "_".join(filename.split("_")[1:-1]))
+    titl = "%s %s" % (" ".join(filename.split("_")[1:-1]), filename.split("_")[0])
+    s = find_file(
+        filename,
+        exp_type=file_location,
+        expno=nodename,
+        postproc=postproc,
+        lookup=lookup_table,
+    )
+    fl.next("%s\nraw data" % titl)
+    # }}}
+    # {{{ Cut off trailing data and phase
+    s = s["t2":(-1e3, 1e3)]
+    fl.image(s)
+    ph0 = s["power", -4].sum("t2")
+    ph0 /= abs(ph0)
+    s /= ph0
+    fl.next("%s\nphased" % titl)
+    fl.image(s)
+    # }}}
+    # {{{ Slice to integration bounds
+    s = s["t2":f_range]
+    fl.next("%s\nsliced to integration bounds" % titl)
+    fl.image(s)
+    # }}}
+    # {{{ Select coherence pathway
+    if "ph2" in s.dimlabels:
+        s = s["ph1", 1]["ph2", 0]
+    else:
+        s = s["ph1", 1]
+    fl.next("%s\nselect coh. pathway - real" % titl)
+    fl.image(s.real)
+    # }}}
+    # {{{ Integrate and normalize --> Enhancement
+    s.integrate("t2")
+    s /= max(s.data.real)
+    # }}}
+    # {{{ Set power axis
+    s.setaxis("power", r_[-9999, array(s.get_prop("meter_powers"))])
+    print("here are the dBm", s.getaxis("power"))
+    s.setaxis("power", lambda x: 1e-3 * 10 ** ((x + measured_vs_actual) / 10.0))
+    print("here are the powers", s.getaxis("power"))
+    s.set_units("power", "W")
+    # }}}
+    # {{{ Simple integral
+    fl.next("%s\nsimple integral" % titl)
+    fl.plot(s["power", :-3], "ko", human_units=False)
+    # (human_units = False because the range for the red points tries to force mW, which is incompatible with W)
+    fl.plot(s["power", -3:], "ro", human_units=False)
+    plt.xlabel("power (W)")
+    plt.ylabel("integrated $^1$H NMR signal")
+    # }}}
+    # {{{ Enhancement
+    s /= s["power", 0]
+    fl.next("%s\nenhancement curve" % titl)
+    fl.plot(s["power", :-3], "ko", human_units=False)
+    fl.plot(s["power", -3:], "ro", human_units=False)
+    plt.xlabel("power (W)")
+    plt.ylabel("enhancement")
+    if save_figs:
+        plt.savefig(
+            "%s_enhancement.png" % outname,
+            transparent=True,
+            overwrite=True,
+            bbox_inches="tight",
+        )
+    # }}}
+    # {{{ Export enhancement data as a csv (if prompted)
+    if export_csv:
+        import pandas as pd
+
+        reenhancementdf = pd.DataFrame(data=s.real.data, columns=["Re[E(p)]"])
+        imenhancementdf = pd.DataFrame(data=s.imag.data, columns=["Im[E(p)]"])
+        enhancementdf = pd.DataFrame(data=s.data, columns=["enhancement"])
+        powerdf = pd.DataFrame(data=s.getaxis("power"), columns=["power"])
+        enhancement = powerdf.join(reenhancementdf)
+        enhancement2 = enhancement.join(imenhancementdf)
+        enhancement3 = enhancement2.join(enhancementdf)
```

Comment:
see if [this](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.from_dict.html) works here, instead